ui/src/config/section/compute.js

```diff
@@ -32,9 +32,9 @@ export default {
       permission: ['listVirtualMachinesMetrics'],
       resourceType: 'UserVm',
       params: () => {
-        var params = {}
+        var params = { details: 'servoff,tmpl,nics' }
         if (store.getters.metrics) {
-          params = { state: 'running' }
```

Comment:
I've removed running as otherwise metrics doesn't return all VMs in the list. One can select VM `state` explicitly in the compute/instance page if they want.

Reply:
just realised I can remove state from the params